index.qmd

```diff
@@ -20,21 +20,27 @@ anim_width <- function(x, width1, width2) {
 }
 ```
 
-```{r fix-me-hack-to-include-fontawesome-css, echo=FALSE}
+```{r fix-me-hack-to-include-fontawesome-css, include=FALSE}
 cat("{{< fa box-open >}}  ")
 ```
 
-### The Open-Source Software Office of the Fred Hutchinson Cancer Center
+## The Open Source Software Office at Fred Hutch
 
-#### Our Mission
+### Our Mission
 
-In recognition of the fact that open source software is the engine of
-biomedical data science, the Data Science Lab (DaSL) created WILDS, the 
-open-source software office of the Fred Hutchinson Cancer Center. The 
-mission of WILDS includes the following:
+Open source software has allowed the biomedical data science community to 
+flourish, therefore to encourage open science practices and to cultivate
+the development of open source software at Fred Hutch, the Data Science Lab 
+(DaSL) created WILDS, the open source software office of the Fred Hutch Cancer 
+Center. WILDS stands for **W**orkflows **I**ntegrating **L**arge **D**ata and 
+**S**oftware, which highlights our focus on people in their day-to-day 
+scientific work. The mission of WILDS includes the following:
 
-- Engage with people developing open-source software across the Cancer
-Consortium.
+- Engage with the Cancer Consortium community about how they can adopt open
+science practices into their own workflows.
+- Educate our community about the critical role that open source software plays
+in biomedical data science practice.
+- Support open-source software developers across the Cancer Consortium.
```

Comment:
```suggestion
- Support open source software developers across the Cancer Consortium.
```
Looks like we're switching from "open-source" to "open source", just want to make sure we catch all of the stragglers, I think there are a couple more below this.